internal/config/settings.go

```diff
@@ -228,40 +284,30 @@ func InitGlobalSettings() error {
 // InitLocalSettings scans the json in settings.json and sets the options locally based
 // on whether the filetype or path matches ft or glob local settings
 // Must be called after ReadSettings
-func InitLocalSettings(settings map[string]interface{}, path string) error {
-	var parseError error
+func InitLocalSettings(settings map[string]interface{}, path string) {
 	for k, v := range parsedSettings {
 		if strings.HasPrefix(reflect.TypeOf(v).String(), "map") {
 			if strings.HasPrefix(k, "ft:") {
 				if settings["filetype"].(string) == k[3:] {
 					for k1, v1 := range v.(map[string]interface{}) {
-						if _, ok := settings[k1]; ok && !verifySetting(k1, reflect.TypeOf(v1), reflect.TypeOf(settings[k1])) {
-							parseError = fmt.Errorf("Error: setting '%s' has incorrect type (%s), using default value: %v (%s)", k, reflect.TypeOf(v1), settings[k1], reflect.TypeOf(settings[k1]))
-							continue
-						}
 						settings[k1] = v1
 					}
 				}
 			} else {
 				g, err := glob.Compile(k)
 				if err != nil {
-					parseError = errors.New("Error with glob setting " + k + ": " + err.Error())
+					// ignore it here, since validateParsedSettings()
+					// already informs about the error after parsing
```

Comment:
I was actually thinking that we might remove the invalid entry from the `parsedSettings` in `validateParsedSettings()`, in line with how we handle other error cases, i.e. ensure that in the end `parsedSettings` is fully valid, instead of leaving this a special case that we need to remember about.

Reply:
You're right, deleting the key is the better/correct solution.